include/ignition/sensors/ImuSensor.hh

```diff
@@ -35,6 +35,23 @@ namespace ignition
   {
     // Inline bracket to help doxygen filtering.
     inline namespace IGNITION_SENSORS_VERSION_NAMESPACE {
+
+    // \brief Reference frames enum
+    // NONE : To be used only when <localization>
+    // reference orintation tag is empty.
+    // ENU (East-North-Up): x - East, y - North, z - Up.
+    // NED (North-East-Down): x - North, y - East, z - Down.
+    // NWU (North-West-Up): x - North, y - West, z - Up.
+    // CUSTOM : The frame is described using custom_rpy tag.
```

Comment:
nit: I would place these doc-strings in line with the enum values (see [sdf/Error.hh](https://github.com/ignitionrobotics/sdformat/blob/sdformat12_12.3.0/include/sdf/Error.hh#L48-L59) for an example of what I mean)

Reply:
Done !